modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -0,0 +1,63 @@
+(link:https://issues.redhat.com/browse/LOG-1100[*LOG-1100*])
+
+* JSON logs can now be forwarded as JSON objects, rather than quoted strings, to either Red Hat's managed Elasticsearch cluster or any of the other supported third-party systems. Additionally, you can now query individual fields from a JSON log message inside Kibana which increases the discoverability of specific logs. For more information, see the link:https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-exported-fields[`ClusterLogForwarder`] documentation.
+(link:https://issues.redhat.com/browse/LOG-785[*LOG-785*])
+
+[id="openshift-logging-5-1-0-bug-fixes"]
+== Bug fixes
+
+* Previously, the `ClusterLogForwarder` CR did not show the `input[].selector` element after it had been created. The current release fixes this issue. When you specify a `selector` in the `ClusterLogForwarder` CR, it remains. Fixing this bug was necessary for link:https://issues.redhat.com/browse/LOG-883[LOG-883], which enables using pod label selectors to forward application log data.
+(link:https://issues.redhat.com/browse/LOG-1338[*LOG-1338*])
+
+* Previously, an update in the cluster service version (CSV) accidentally introduced resources and limits for the OpenShift Elasticsearch operator container. Under specific conditions, this caused an out-of-memory condition that terminated the Elasticsearch operator pod. The current release fixes this issue by removing the CSV resources and limits for the operator container. Now, the operator gets scheduled without issues.
+(link:https://issues.redhat.com/browse/LOG-1254[*LOG-1254*])
+
+* Previously, forwarding logs to Kafka using chained certificates failed with the following error message:
++
+`state=error: certificate verify failed (unable to get local issuer certificate)`
++
+Logs could not be forwarded to a Kafka broker with a certificate signed by an intermediate CA. This happened because the fluentd Kafka plugin could only handle a single CA certificate supplied in the `ca-bundle.crt` entry of the corresponding secret. The current release fixes this issue by enabling the fluentd Kafka plugin to handle multiple CA certificates supplied in the `ca-bundle.crt` entry of the corresponding secret. Now, logs can be forwarded to a Kafka broker with a certificate signed by an intermediate CA.
+(link:https://issues.redhat.com/browse/LOG-1218[*LOG-1218*], link:https://issues.redhat.com/browse/LOG-1216[*LOG-1216*])
+
+* Previously, while under load, Elasticsearch responded to some requests with an HTTP 500 error, even though there was nothing wrong with the cluster. Retrying the request was successful. This release fixes the issue by updating the cron jobs to be more resilient when they encounter temporary HTTP 500 errors. The updated cron jobs will first retry a request multiple times before failing.
```

Comment:
We should be more precise when we speak about cronjobs as `index management cronjobs`.

Reply:
Done.